core/src/mender-deployment-data.c

```diff
@@ -279,6 +282,41 @@ __mender_deployment_data_set_string(mender_deployment_data_t *deployment_data, c
     return MENDER_OK;
 }
 
+mender_err_t
+__mender_deployment_data_get_state(mender_deployment_data_t *deployment_data, const char *key, mender_update_state_t *state) {
```

Comment:
Why do we need `key`? Are we ever going to have multiple states under different keys?

Reply:
We don't need it, residue from the definition of `__mender_deployment_data_set_string`